TaskfMRIAnalysis/TaskfMRIAnalysis.sh

```diff
@@ -52,9 +52,10 @@ then
     export HCPPIPEDIR="$(dirname -- "$0")/.."
 fi
 
+source "$HCPPIPEDIR/global/scripts/opts.shlib"			# Function for getting HCPpipelines version
```

Comment:
We should almost certainly not mix opts.shlib and newopts.shlib.  I don't see other changes to this file in this commit that use it, so I don't understand why it was sourced.

As for the comment on the line, the main way to get pipeline version info is now by running `"$HCPPIPEDIR"/show_version`.  It also has a `--short` option.  The other existing methods have been refitted to run that script one way or another.  I have not put a hardcoded `--version` option into newopts.shlib (while `--help` and `-h` are hardcoded), this could be added if it is desired to have all scripts return the overall pipelines version info (which seems slightly silly, as the user can run `show_version` instead of using some other script).